mux/scripts/smoke-tui.py

```
@@ -0,0 +1,105 @@
+BIN = os.environ.get("CMUX_MUX_BIN", "target/debug/cmux-mux")
+SESSION = f"smoke-{os.getpid()}"
+SOCK = os.path.join(os.environ.get("TMPDIR", "/tmp"), f"cmux-mux-{os.getuid()}", f"{SESSION}.sock")
+
+def rpc(cmd):
+    s = socket.socket(socket.AF_UNIX)
+    s.connect(SOCK)
+    s.sendall((json.dumps(cmd) + "\n").encode())
+    buf = b""
+    while not buf.endswith(b"\n"):
+        chunk = s.recv(65536)
+        if not chunk: break
```

Ruff (0.15.20)

[error] Multiple statements on one line (colon)

(E701)